### PR TITLE
Add :zachL: and :zachR: reaction to Toxic_Z's messages

### PR DESCRIPTION
**Pull request by zimmerry**
As requested by Erick.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -118,6 +118,14 @@
   event.message.react '🙏' if event.author.roles.any? { |r| r.id == CONFIG['roles']['god'] }
 end
 
+# add :zachR: and :zachL: react to Toxic_Z
+bot.message do |event|
```
Comment (detjensrobert) [inline main.rb:122]: This block can be combined with the previous block.

**Review by detjensrobert — CHANGES_REQUESTED** [review]
the mad man actually did it

**Review by detjensrobert — APPROVED** [review]
Not sure if the custom emoji should just be the ID string, but I can update that if it is a problem

**Comment by zimmerry** [comment]
> Not sure if the custom emoji should just be the ID string, but I can update that if it is a problem

I tested it, it worked with a custom emoji I added to my test server.

**Pull request merged**